modules/installation-configuration-parameters.adoc

```diff
@@ -482,7 +482,7 @@ in vSphere.
 |String.
 
 |`platform.vsphere.folder`
-|_Optional_. The absolute path of an existing folder where the installation program creates the virtual machines. create VMs. If you do not provide this value, the installation program creates a folder that is named with the cluster ID is created in the datacenter virtual machine folder.
+|_Optional_. The absolute path of an existing folder where the installation program creates the virtual machines. If you do not provide this value, the installation program creates a folder that is named with the infrastructure ID in the datacenter virtual machine folder.
```

Comment:
This is only true for IPI. For UPI I would suggest:

"The absolute path to the virtual machine folder. The default value is a top-level folder named with the infrastructure ID.`

But I think this is shared beween IPI and UPI so not sure how to do that. My suggestion would technically be true for both cases, but is a little less informative for IPI.

Reply:
I think most of the platform fields in install config was only valid for IPI because UPI is just pausing before IPI and then doing what the installer would have done ourselves because that allows user to do things differently. So i'm fine if this most accurate for IPI.